apps/condo/domains/common/hooks/useMultipleFiltersModal.tsx

```diff
@@ -333,6 +333,7 @@ const ResetFiltersModalButton: React.FC<ResetFiltersModalButtonProps> = ({
             type={'text'}
             onClick={handleReset}
             size={size}
+            data-cy={'filters-button-reset'}
```

Comment:
It’s better to use something like ‘<domain>__<component name>—<element>’ format for such cases

Reply:
Refactored this